toolchain/lc-compile/src/set.c

```diff
@@ -22,7 +22,7 @@ typedef unsigned int SetRef;
 
 #define MAX_SET_DEPTH 256
 SetRef s_sets[MAX_SET_DEPTH];
-unsigned int s_set_count = 0;
```

Comment:
@peter-b I'm not sure why this should be signed here. Happy to be educated if you have the time. The rest of this looks ok for review to me but I don't want to review something ok when I don't understand it. After reviewing the code in context I'm even more perplexed the the APIs use signed parameters so maybe I'd better leave this review to someone that knows this bit of the platform.